src/main/java/com/coveros/selenified/OutputFile.java

```diff
@@ -575,10 +575,12 @@ public void finalizeOutputFile() {
         replaceInFile("STEPSPERFORMED", Integer.toString(fails + passes));
         replaceInFile("STEPSPASSED", Integer.toString(passes));
         replaceInFile("STEPSFAILED", Integer.toString(fails));
-        if (fails == 0) {
-            replaceInFile("PASSORFAIL", "<font size='+2' class='pass'><b>PASS</b></font>");
+        if (fails == 0 && errors == 0 && testStatus == 1) {
+            replaceInFile("PASSORFAIL", "<font size='+2' class='pass'><b>SUCCESS</b></font>");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "PASSORFAIL" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)